openapi/kas-fleet-manager.yaml

```diff
@@ -1409,10 +1409,6 @@ components:
               description: "This field is now deprecated, please use the /api/kafkas_mgmt/v1/instance_types/{cloud_provider}/{cloud_region} endpoint to retrieve the field instead."
             reauthentication_enabled:
               type: boolean
-            kafka_storage_size:
```

Comment:
Would have been good to bump the API version as well over here https://github.com/valeriiashapoval/kas-fleet-manager/blob/2b86e31a2fcea4b28dc7f0bd4e3708f38d3d558b/openapi/kas-fleet-manager.yaml#L5.

FYI @valeriiashapoval